enarx-pr-assign

```diff
@@ -40,14 +75,14 @@ def get_responsible(pr):
     # assign responsibility. Still others modify the effects of earlier events.
     # The last event which assigns responsibility is the one that we choose.
 
-    events = merge_events_and_reviews(pr)
+    events = sorted(merge_events_and_reviews(repo, pr), key=lambda x: x["when"])
     participants = {x["who"] for x in events} - {pr.user.login}
     segregated = {u: [x["what"] for x in events if x["who"] == u] for u in participants}
 
     for (who, whats) in segregated.items():
         unrequested = False
         for what in reversed(whats):
-            if what == "REVIEW_REQUESTED" and not unrequested:
+            if what == "ReviewRequestedEvent" and not unrequested:
                 yield who
             elif what in ("DISMISSED", "PENDING"):
```

Comment:
Did this not change to lowercase?

Reply:
Yes, this is correct. The previous version yielded everything in all caps; this version yields data directly from Github. Pull request events (like `ReviewRequestedEvent`) are first-letter capitalized, as changed above, but PR review states (like `DISMISSED`, `PENDING`, `APPROVED`) are all caps.